Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-Rapid_Breach_Response_-_Set_Incident_Info_README.md

```diff
@@ -0,0 +1,34 @@
+This playbook is responsible for setting up the Rapid Breach Response Incident Info tab.
```

Comment:
The playbook is responsible for setting up the Rapid Breach Response Incident Info tab **in the layout**.